### PR TITLE
Open search shortcut

### LINKED ISSUE #1213: Introduce Shortcut / to focus search
As [Sebastian](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#sebastian---tech-lead) and [Mathew](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#matthew---cloud-native-software-engineer) I would like to be able use shortcuts `/` to focus on the search field
**Acceptance Criteria**

- [ ] Follow Project DOD
- [ ] We target first list views - stretch goal is to cover detail views too
- [x] typing `/` will focus on the first search field

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**
As a user, after clicking `/` on the keyboard I want to open a search input.
Changes proposed in this pull request:
- Added listeners for `/`
- Changed search input for CustomResources
- ...

**Related issue(s)**
Closes #1213

### CODE REVIEW

File: core-ui/src/resources/createResourceRoutes.js [hunk 1]
```diff
@@ -30,6 +34,7 @@ export const usePrepareListProps = (resourceType, resourceI18Key) => {
     resourceName: resourceI18Key ? t(resourceI18Key) : '',
     namespace: routerParams.namespaceId,
     i18n,
+    allowSlashShortcut,
```
Comment (dariadomagala) [inline core-ui/src/resources/createResourceRoutes.js:37]: why don't we always just set it to true here?

Comment (mrCherry97) [reply core-ui/src/resources/createResourceRoutes.js:37]: we set it in line 67 
```
const props = usePrepareListProps(resourceType, resourceI18Key, true);
```

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**